src/app_service/common/activity_center.nim

```diff
@@ -1,8 +1,16 @@
-import json
-
+import json, json_serialization
 import ../../app/core/eventemitter
 
+include json_utils
+
 const SIGNAL_PARSE_RAW_ACTIVITY_CENTER_NOTIFICATIONS* = "parseRawActivityCenterNotifications"
 
 type RawActivityCenterNotificationsArgs* = ref object of Args
   activityCenterNotifications*: JsonNode
+
+proc checkAndEmitACNotificationsFromResponse*(events: EventEmitter, result: JsonNode) =
+  if result{"activityCenterNotifications"} == nil or result{"activityCenterNotifications"}.kind == JNull:
```

Comment:
Could be good to simplify this expression to avoid retyping result{"activityCenterNotification"}, maybe using the get(...) and avoid comparing to nil.

Reply:
Good advice and it made me think that it's better to pass `result{"activityCenterNotifications"}` to the function, since we can't be sure yet that all responses will have notifications by that name. If you have concerns about this approach, let's open a ticket and discuss it, and I think there is still a lot we can fix. 

About nil and JNull checks. This procedure is used in several places, we have no control because AC notifications here can be either nil or null-object (even if this response is constructed from status-go json).